lib/src/providers/ethereum_walletconnect_provider.dart

```diff
@@ -33,6 +33,24 @@ class EthereumWalletConnectProvider extends WalletConnectProvider {
     return result;
   }
 
+  /// Signs method calculates an Ethereum specific signature.
+  /// [address] - 20B address
+  /// [message] - message to sign
+  /// [password] - The password of the account to sign data with
+  ///
+  /// Returns signature.
+  Future<String> pseronalSign(
```

Comment:
function name should be `personalSign` instead of `pseronalSign`

Reply:
Thanks for the PR. Please update the function name and I'll merge it!